conf/server.yaml.template

```diff
@@ -1,6 +1,29 @@
 SERVER:
-  # Server hostname
-  # HOSTNAME: replace-with-satellite-hostname
+  # Server hostnames, a list of one or more dns-resolvable hostnames
+  HOSTNAMES:
+    - null
+  #  - replace.with.satellite.hostname
+  #  - replace.with.satellite.hostname
+  VERSION:
+    # The full release version (6.9.2)
+    # RELEASE: populate with satellite version
+    # The snap version currently testing (if applicable)
+    # SNAP:
+    # The source of Satellite packages. Can be one of:
+    # internal, ga, beta
+    SOURCE: "internal"
+  # run-on-one - All xdist runners default to one satellite
+  # balance - xdist runners will be split between available satellites
+  # on-demand - any xdist runner without a satellite will have a new one provisioned.
+  XDIST-BEHAVIOR: "run-on-one"
```

Comment:
I believe all other dynaconf yamls have been `_` delineated, and we should stay consistent.

Reply:
Interestingly, dynaconf seems to convert this automatically anyway. hmmmm I'm not as sure that we need to be consistent.

Reply:
easy enough

Reply:
fixed